calculator.js

```diff
@@ -0,0 +1,19 @@
+
+//trying to build a calculator
+
+var x;
+var y;
```

Comment:
You do not need lines 4 and 5 here. We will talk about why in our Scope and Closures class tomorrow. Otherwise great job on this assignment! You should consider tackling the bonus!